src/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java

```diff
@@ -49,5 +88,29 @@ else if (reason.equals("transfer")){
 			gm.invalidateCache(g.getName());
 		}
 	}
+	
+	@EventHandler(priority = EventPriority.HIGHEST)
+	public void onLogin(PlayerJoinEvent event){
+		MercuryPlugin.handler.sendMessage("all", "name_layer", "login "+event.getPlayer().getDisplayName());
+		Bukkit.getLogger().info("LOGIN");
+	}
+	
+	@EventHandler(priority = EventPriority.HIGHEST)
+	public void onLogoff(PlayerQuitEvent event){
+		MercuryPlugin.handler.sendMessage("all", "name_layer", "logoff "+event.getPlayer().getDisplayName());
+		Bukkit.getLogger().info("LOGOFF");
+	}
 
+	@Override
+	public void run() {
+		NameLayerPlugin nl = NameLayerPlugin.getInstance();
+		nl.isMercuryEnabled = Bukkit.getPluginManager().isPluginEnabled("Mercury");
+		if (nl.isMercuryEnabled){
+			nl.getServer().getPluginManager().registerEvents(this, nl);
```

Comment:
I was actually being rushed out the house and trying to get you sorted lol.
Guh NameLayer master has so many changes to live.
On Jun 5, 2015 12:27 PM, "Darius Scott" <notifications@github.com> wrote:

> In src/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java
> <https://github.com/Civcraft/NameLayer/pull/112#discussion_r31827926>:
>
> >
> > +	@Override
> > +	public void run() {
> > +		NameLayerPlugin nl = NameLayerPlugin.getInstance();
> > +		nl.isMercuryEnabled = Bukkit.getPluginManager().isPluginEnabled("Mercury");
> > +		if (nl.isMercuryEnabled){
> > +			nl.getServer().getPluginManager().registerEvents(this, nl);
>
> Rourke go back to sleep lol
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Civcraft/NameLayer/pull/112/files#r31827926>.
>